scripts/build/command.mjs

```diff
@@ -35,9 +37,15 @@ async function run() {
     packages: 'external',
     outdir: `${pkgPath}/dist`,
     entryPoints: [entryPoint],
+    metafile: true,
   });
-  await Promise.all([typeCheckingTask, esbuildTask]);
+  const result = await Promise.all([typeCheckingTask, esbuildTask]);
   console.log(`[build] ${path} built successfully.`);
+
+  await fs.writeFile(
+    `dist/${packageName}.build.json`,
```

Comment:
You are harcoded `.build.json` here and other functions. please make all of them to be generated from same place to make sure it can be easily changed and understand in future.